types/src/data_types.rs

```diff
@@ -66,6 +66,11 @@ pub enum DataTypeNode {
     /// Precision and optional timezone
     DateTime64(DateTimePrecision, Option<String>),
 
+    /// Optional timezone
+    Time(Option<String>),
+    /// Precision and optional timezone
+    Time64(DateTimePrecision, Option<String>),
```

Comment:
But it does not have a timezone?
* https://clickhouse.com/docs/sql-reference/data-types/time
* https://clickhouse.com/docs/sql-reference/data-types/time64